docs/DEPLOY.md

```diff
@@ -1 +1,104 @@
-## Deployment
\ No newline at end of file
+## Deployment
+
+### Creating an AWS Account
+
+  - [Step by step guide](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/)
+
+### Create IAM Roles
+
+  - [Follow steps 3-4](https://docs.aws.amazon.com/codedeploy/latest/userguide/getting-started-provision-user.html)
+
+### Create and Launch an EC2 Instance
+
+  - Choose *Amazon Linux 2 AMI (HVM) - Kernel 5.10, SSD Volume Type* as your machine image
+  - Choose *t2.micro* (Free tier eligible) for instance type
+  - Under Configure Instance Details, select the previously created IAM Role.
+  - Under Configure Security Group, Allow incoming access to:
+ 
+| Type         | Port Range | Description.                                |
+| ------------ | :--------: | ------------------------------------------: |
+| SSH          | 22         |  Port for SSH into server                   |
+| HTTP         | 80         |  Port for HTTP requests to web server       |
+| HTTPS        | 443        |  Port for HTTPS requests to web server      |
+| Custom TCP   | 5000       |  Port for api                               |
+
+  - Finally, launch the instance and create a new key pair and store it in a secure and accessible location. (This will generate a .pem key which is used for SSH)
+
+### Install Packages in EC2 Instance
+
+  - Locate your EC2 Instance under Instances on the EC2 page.
+  - Locate your Public IPv4 DNS, and SSH into it using ur .pem key (ssh -i "file.pem" ec2-user@xxxxxxxx.compute-1.amazonaws.com
```

Comment:
your*, fix typo.

Reply:
^^ your not ur

FYI @stalhaali you can highlight one or more lines and click to add a suggestions then make some edits - makes it a bit easier for the person 😄